part-08-structure-and-versioning/app/api/api_v1/api.py

```diff
@@ -1,6 +1,6 @@
 from fastapi import APIRouter
 
-from app.api.api_v1.endpoints import recipe
+from api.api_v1.endpoints import recipe
```

Comment:
prefer absolute imports like `app.api.api_v1.endpoints`. If it's not working it's because you need to set your PYTHONPATH